bolt.gemspec

```diff
@@ -45,6 +45,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "puppet", [">= 6.0.1", "< 7"]
   spec.add_dependency "puppet-resource_api"
   spec.add_dependency "r10k", "~> 2.6"
+  spec.add_dependency "ruby_smb", "~> 1.0"
```

Comment:
Will have to add component to bolt-vanagon. I guess https://rubygems.org/gems/ruby_smb/versions/1.0.5 should be good for now? https://github.com/puppetlabs/bolt-vanagon/tree/master/configs/components

Reply:
This has already been approved as part of https://tickets.puppetlabs.com/browse/RE-12026

Reply:
PR up at https://github.com/puppetlabs/bolt-vanagon/pull/80